test/service_instance_and_binding_test/service_binding_test/service_binding_test.go

```diff
@@ -398,6 +405,89 @@ var _ = DescribeTestsFor(TestCase{
 							})
 						})
 
+						Context("Instance Sharing - when binding a reference instance", func() {
```

Comment:
rename to When("binding a reference instance") add the instance sharing context wrapper before this when